src/service/dynamo/dynamo.submission.service.ts

```diff
@@ -0,0 +1,30 @@
+import { PutItemCommand, DynamoDBClient } from '@aws-sdk/client-dynamodb';
+import { marshall } from '@aws-sdk/util-dynamodb';
+import { log } from '../../utils/logger';
+import * as config from '../../config';
+import { isFeatureEnabled } from '../../utils/isFeatureEnabled';
+import { getUserEmail } from '../../utils/getUserEmail';
+
+const client = new DynamoDBClient({
+    region: config.REGION,
+    endpoint: config.DYNAMO_ENDPOINT
+});
+
+export const putSubmission = async (id: string, jwt: string, appData: any) => {
+
+    if (isFeatureEnabled(config.FEATURE_FLAG_ENABLE_DYNAMO)) {
+
+        const params = { TableName: config.DYNAMO_TABLE_NAME,
+            Item: marshall({ id: id, data: { ...appData, submission_email_address: getUserEmail(jwt) } })
```

Comment:
I will assume that `getUserEmail` will fail if **JWT** is not set correctly. We should return a mock email if it is not a production environment.
We are going to propagate the email through `res.locals`, so the variable will be accessible in templates to be rendered and throughout the request object's journey. However, this is a task for another time. I will create the related ticket for it.

Reply:
Yea you are right, it will fail if JWT isn't set correctly at the moment. I could add some inline logic to render the mock email then? (Or, make a util `isProduction` as we have production check logic repeated in `authentication.middleware.ts`?)

```
submission_email_address: config.NODE_ENV === 'production ? getUserEmail(jwt) : 'fakeemail@fake.com 
```

Eventually, we will migrate to the `res.locals` solution as discussed in standup

Reply:
As agreed in stand-up, logic will be replaced with `res.locals` solution so for now conditional check will be implemented with ternary.